src/structures/Role.js

```diff
@@ -205,9 +205,10 @@ class Role {
     return this.client.api.guilds[this.guild.id].roles[this.id].patch({
       data: {
         name: data.name || this.name,
```

Comment:
If you are re-ordering it alphabetically, isn't name supposed to be before `position`?